### PR TITLE
Update undergrad-research.md

### PR DESCRIPTION
**Pull request by rachitnigam**
Require more answers from senior students

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/undergrad-research.md [hunk 1]
```diff
@@ -32,9 +36,10 @@ We are thrilled that you're interested in research in our research group! Please
 (It's OK to say, "I don't know; I'm looking to explore!")
 
 
-## Optional Stuff
+## Background
 
-(Feel free to leave any of these blank.)
+- **Freshmen or Sophomore**: The following are optional.
+- **Junior or above**: The first two questions are required, others are optional.
```
Comment (sampsyo) [inline .github/ISSUE_TEMPLATE/undergrad-research.md]: I think this is a good idea. We can be forgiving with the answers but getting people to at least try to find an interesting paper would be useful.

Here's an alternative wording that has the bonus of avoiding the weirdly gendered name for first-year college students. :smiley:

> If you're in your first or second year at Cornell, these are all optional. If you're a junior or senior, please answer the first two (the rest is still optional).

File: .github/ISSUE_TEMPLATE/undergrad-research.md [hunk 2]
```diff
@@ -19,6 +21,8 @@ We are thrilled that you're interested in research in our research group! Please
 
 **Relevant classes**:
 
+**Willing to do research during the summer:**
```
Comment (sampsyo) [inline .github/ISSUE_TEMPLATE/undergrad-research.md]: We kinda already ask this below under "When do you want to do research?" Maybe we can clarify there that people should say, even if they want to get started during the semester, that they should answer whether they're interested in _continuing_ into the summer.

File: .github/ISSUE_TEMPLATE/undergrad-research.md (hunk 1 above)
Comment (rachitnigam) [reply .github/ISSUE_TEMPLATE/undergrad-research.md]: Addressed

File: .github/ISSUE_TEMPLATE/undergrad-research.md (hunk 2 above)
Comment (rachitnigam) [reply .github/ISSUE_TEMPLATE/undergrad-research.md]: Addressed

**Pull request merged**